config/webpack.dev.config.js

```diff
@@ -49,12 +49,17 @@ module.exports = {
         postcss: [autoprefixer, precss],
       }
     })
-  ],
-  resolve: {
+  ]
+}
+
+if (process.env.BABEL_ENV === 'TEST') {
+  devConfigs.revolve = {
```

Comment:
I see, I looked at this file and saw that the alias was added after being commented out in this commit: https://github.com/jpuri/react-draft-wysiwyg/commit/63d6a8a4e5dafed5f57987120f4d1535edcb27c1

I didn't want to break the tests while fixing it so I went about it this way.